packages/web-api/src/WebClient.ts

```diff
@@ -613,3 +614,25 @@ function warnDeprecations(method: string, logger: Logger): void {
     logger.warn(`${method} is deprecated. Please check on https://api.slack.com/methods for an alternative.`);
   }
 }
+
+/**
+ * Log a warning when using chat.postMessage without text argument
+ * @param method api method being called
+ * @param logger instance of we clients logger
+ */
+function warnMissingTextArgument(method: string, logger: Logger, options?: WebAPICallOptions): void {
+  const methodsWithOptionalText = ['chat.postEphemeral', 'chat.postMessage', 'chat.scheduleMessage', 'chat.update'];
+  const isNeededToCheckMethod = methodsWithOptionalText.includes(method);
+
+  const isEmptyText = (args: WebAPICallOptions) =>
```

Comment:
Sorry, the implementation I mentioned had lack of consideration for `attachments`. Can you update this logic to check `fallback` in the case of `attachments`? Refer to https://github.com/slackapi/python-slack-sdk/pull/966 for the logic.